eole/inputters/inputter.py

```diff
@@ -90,7 +90,7 @@ def _read_vocab_file(vocab_path, min_count):
         raise RuntimeError("Vocabulary not found at {}".format(vocab_path))
     else:
         with codecs.open(vocab_path, "rb") as f:
-            lines = [line.decode("utf-8") for line in f.read().split(b"\n")]
+            lines = [line.decode("utf-8") for line in f.read().replace(b"\r", b"").split(b"\n")]
```

Comment:
I don't think we can do that.
Some vocabs like llama have this strange "\r" as a token or even "\r" added" to another character, why ? I don't know but this change will break things.
cc @francoishernandez

Reply:
@ArtanisTheOne the "\r" you need to remove here is supposed to always be at the end of the line, so you can probably adapt the logic to handle the aforementioned case, e.g. by capturing only "\r" that are not at the beginning of a line.

Reply:
@ArtanisTheOne is this happening with your own vocab ? if so please post it in the issue, if not is this the llama one?

Reply:
yes because exactly those specific cases where special line breaks would mess up things so we had to fallback to "\n" split only